src/components/issues/ActionForm.tsx

```diff
@@ -108,15 +108,15 @@ const ActionForm: FC<ActionFormProps> = ({
     const handleCreateTask = async (e: MouseEvent) => {
         e.preventDefault();
         setIsLoading(true);
-        await createTask(state);
+        if (createTask) await createTask(state);
         setIsLoading(false);
     };
 
     const handleUpdateTask = async (e: MouseEvent) => {
         e.preventDefault();
         setIsLoading(true);
         setShowSuggestion(false);
-        await updateTask(state, taskId);
+        if (updateTask) await updateTask(state, taskId);
```

Comment:
When is updateTask function worked here??

Reply:
It calls the update task function passed from the parent element.